Tests/conf.json

```diff
@@ -2937,6 +2937,11 @@
         {
             "integrations": "Imperva WAF",
             "playbookID": "Imperva WAF - Test"
+        },
+        {
+            "playbookID": "FailedInstances - Test",
+            "integrations": "Carbon Black Enterprise EDR",
+            "fromversion": "5.0.0"
```

Comment:
why is the test fromversion 5.0 ?

Reply:
Cause the test is using CB Enterprise EDR and it from 5.0.0

Reply:
so lets configure an integration which is not fromversion 5.0
the script is fromversion 4.0, so need to be tested on all versions